src/main.h

```diff
@@ -107,6 +107,8 @@ static const bool DEFAULT_TXINDEX = false;
 static const unsigned int DEFAULT_BANSCORE_THRESHOLD = 100;
 
 static const bool DEFAULT_TESTSAFEMODE = false;
+/** Default for -permitrbf */
+static const bool DEFAULT_PERMIT_REPLACEMENT = true;
```

Comment:
@kristovatlas This is not a consensus rule, so you're free to choose a
different setting than the default here.

As I've said on the PR being linked to, I'm certainly willing to consider
changing the default if there is controversy about it. But can someone at
least point out one service or wallet that is not dealing with this
correctly and has no plans to do so in the near future?